pkg/ansible/controller/controller.go

```diff
@@ -35,14 +35,12 @@ import (
 
 // Options - options for your controller
 type Options struct {
-	EventHandlers []events.EventHandler
-	LoggingLevel  events.LogLevel
-	Runner        runner.Runner
-	Namespace     string
-	GVK           schema.GroupVersionKind
-	// StopChannel is used to deal with the bug:
-	// https://github.com/kubernetes-sigs/controller-runtime/issues/103
-	StopChannel <-chan struct{}
+	EventHandlers   []events.EventHandler
+	LoggingLevel    events.LogLevel
+	Runner          runner.Runner
+	Namespace       string
```

Comment:
Sorry, one more question. Does this actually get used anymore? It looks like it was only used in the reconcile loop.